src/main/resources/messages.properties

```diff
@@ -0,0 +1,3 @@
+validation.doctorID.mandatory=doctorID is mandatory.
+validation.patientID.mandatory=patientID is mandatory.
+db.collection.name=doctor_patient_assignment
```

Comment:
why collection name is in messages.properties
It should be in constants

Reply:
okay, changed it.